src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageParserTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.google.common.collect.ImmutableList;
+import com.microsoft.azure.servicebus.MessageBody;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.exceptions.InvalidMessageException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+import static com.microsoft.azure.servicebus.MessageBody.fromBinaryData;
+import static com.microsoft.azure.servicebus.MessageBody.fromSequenceData;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer.SamplePaymentMessageData.paymentMessage;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer.SamplePaymentMessageData.paymentMessageJsonAsByte;
+
+
+public class PaymentMessageParserTest {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private PaymentMessageParser paymentMessageParser;
+
+    @BeforeEach
+    public void before() {
+        paymentMessageParser = new PaymentMessageParser(objectMapper);
```

Comment:
We can just build a static final instead

Reply:
done